### PR TITLE
[Fix] fix compilation error when setting USE_RELAY_DEBUG

### PR DESCRIPTION
**Pull request by xutianming**
This pull request fixes a compilation error when setting  USE_RELAY_DEBUG.
`InstructionPrint` has been moved to [bytecode.cc](https://github.com/apache/incubator-tvm/blob/master/src/runtime/vm/bytecode.cc#L500)

### CODE REVIEW

**Comment by xutianming** [comment]
please check / test / retest this

**Comment by xutianming** [comment]
@icemelon9 Could you please review this PR ?

**Comment by xutianming** [comment]
CC @zhiics @tqchen @jroesch

**Review by zhiics — COMMENTED** [review]
I think we can just remove these three lines

File: src/runtime/vm/vm.cc [hunk 1]
```diff
@@ -340,7 +340,7 @@ void VirtualMachine::RunLoop() {
     auto const& instr = code_[this->pc_];
     DLOG(INFO) << "Executing(" << pc_ << "): " << instr;
 #if USE_RELAY_DEBUG
-    InstructionPrint(std::cout, instr);
+    LOG(INFO) << instr;
 #endif  // USE_RELAY_DEBUG
```
Comment (zhiics) [inline src/runtime/vm/vm.cc:344]: we can just remove them

Comment (xutianming) [reply src/runtime/vm/vm.cc]: Done

**Review by zhiics — APPROVED** [review]

**Comment by zhiics** [comment]
Thanks @xutianming

**Pull request merged**